tests/e2e/tests/admin/signup.spec.ts

```diff
@@ -96,8 +96,11 @@ test.describe('Sign Up', () => {
   test('a user should be able to signup when the strapi instance starts fresh', async ({
     page,
   }) => {
-    await page.getByRole('button', { name: "Let's start" }).click();
+    await Promise.all([
+      page.waitForLoadState('networkidle'),
```

Comment:
Maybe we should make networkidle a const somewhere, it gets used quite a bit

Reply:
Since it's a typed value for waitForLoadState I'll leave it for now, but I actually think the next time I'm in here messing with things I'm going to add a util for navigating + waiting so you can just call one line instead of 2+, nobody should have to remember to do this to write a stable test